service/apiserver/internal/logic/nft/getaccountnftslogic.go

```diff
@@ -70,21 +70,25 @@ func (l *GetAccountNftsLogic) GetAccountNfts(req *types.ReqGetAccountNfts) (resp
 		return resp, nil
 	}
 
-	nftList, err := l.svcCtx.NftModel.GetNftListByAccountIndex(accountIndex, int64(req.Limit), int64(req.Offset))
+	nfts, err := l.svcCtx.NftModel.GetNftListByAccountIndex(accountIndex, int64(req.Limit), int64(req.Offset))
 	if err != nil {
 		return nil, types2.AppErrInternal
 	}
 
-	for _, nftItem := range nftList {
+	for _, nft := range nfts {
+		creatorName, _ := l.svcCtx.MemCache.GetAccountNameByIndex(nft.CreatorAccountIndex)
```

Comment:
Can we handle this right when creator account index is -1?

Reply:
yes, it will return nil.